server/aws-lsp-codewhisperer/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [0.0.5] - 2024-05-15
+- Create Chat server export it for consumption
+- Fix duplicate hover message in security scan
+- Clear security scan finding when a project is unloaded
+- Fix the consumption of streaming client
+- Add `json`, `yaml` and `java` to list of supported languages
```

Comment:
Please make sure [this PR](https://github.com/aws/language-servers/pull/239) is also included. I expect it to be merged by tomorrow morning.